pyartcd/pyartcd/pipelines/operator_sdk_sync.py

```diff
@@ -25,9 +26,20 @@ def __init__(self, runtime: Runtime, group: str, assembly: str, nvr: str, prerel
         self.parent_jira_key = ""
         self._jira_client = runtime.new_jira_client()
         self.arches = arches
+        self.force = force
 
     async def run(self):
-        if self.assembly:
+        if self.nvr:
+            build = koji.ClientSession(constants.BREW_SERVER).getBuild(self.nvr)
+        else:
+            if not self.assembly:
```

Comment:
if triggered by scheduled job the assembly value will be none, and we check Cincinnati candidate channel file which is updated by promote job run to get the assembly, it has the latest in-building version at the end of its version list